### PR TITLE
Feature to create new build from roger itself

### PR DESCRIPTION
**Pull request by shidhincr**
WHY ??

Because I couldn't find a way to restart the build for a tag I created today.

### CODE REVIEW

**Comment by razanbilwani** [comment]
image

File: src/client/components/projectDetails.jsx [hunk 1]
```diff
@@ -15,6 +15,29 @@ class ProjectDetails extends React.Component {
...
+  customBuild(build){
+    let path = window.prompt(`Enter the name of the branch/tag you need to build.
+
+      Format:
+
+        "Any repo: Any branch/ tag"
+
+      Examples:
+
+        raven: latest
```
Comment (razanbilwani) [inline src/client/components/projectDetails.jsx]: @shidhincr are you sure 'latest' works?, it has to be branch 'master' generally

Comment (shidhincr) [inline src/client/components/projectDetails.jsx]: Roger handles it.. here 👇
 https://github.com/namshi/roger/blob/master/src/client/actions/actions.js#L11

Comment (razanbilwani) [inline src/client/components/projectDetails.jsx]: :+1: awesome

**Comment by odino** [comment]
@ashiddo11 can we test this out in our internal infra?

**Pull request merged**